miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniApp.kt

```diff
@@ -69,6 +69,12 @@ abstract class MiniApp internal constructor() {
         RequiredPermissionsNotGrantedException::class
     )
     @Suppress("LongParameterList", "LongMethod")
+    @Deprecated(
```

Comment:
@gopi-karmakar , We are not changing any existing logic for these interfaces as of now. The reason why we have mentioned them as `@Deprecated` is to notify the host app that these interfaces won't be available in our future versions.

Reply:
@munir-rakuten @rleojoseph Understood, Well, In that case moving all the `create` and `createWithUrl` APIs to `MiniApp.kt` makes more sense. Since the existing `create` APIs in the `MiniApp.kt` will be removed in the future and should be replaced by the new `create` & `createWithUrl` apis to create multiple `MiniApp` instances.

Reply:
Just to clearify the change is correct @munir-rakuten the `MiniApp.create` method will be deprecated and replaced by `MiniAppView`'s `load` method

```
// old
// MiniApp.create() { ... }

// new
let view = MiniAppView()
view.load() { // this will then tell MiniAppViewHandler to load all necessary things
    result in
    //... 
} 
```

Reply:
@laubengaier 
Alright, By the above sample code it's making clear that we're not maintaining the list `MiniAppView` instances to the SDK rather we are expecting the `HostApps` to maintain the list of `MiniAppView` instances and every host app should do all that tedious work rather the SDK doing it once for all the `HostApps` and `HostApps` just reuse the code by simply calling the APIs and now I see, that's why it feels like there's nothing `Manager` needed on top of `MiniAppView` instances. 

Well  I've understood. Well, it may be solving the purpose now but I feel later we may eventually need maintain the list of `MiniAppView` instances in the SDK only when many new features will be added for e.g: `clearMiniApps()` or `clearMiniApp(miniAppId)` 

However, Now I am fine to refactor the SDK again in the future as per the requirements. So It doesn't matter anymore now.